pkg/controller/common/util.go

```diff
@@ -158,6 +158,8 @@ func GetMeshNamespaces(controlPlaneNamespace string, smmr *v1.ServiceMeshMemberR
 	}
 	meshNamespaces := sets.NewString(controlPlaneNamespace)
 	if smmr != nil {
+		// XXX: I don't think this can work, since ConfiguredMembers won't be set
+		// until after the initial install succeeds
 		meshNamespaces.Insert(smmr.Status.ConfiguredMembers...)
```

Comment:
Why not use `SMMR.Spec.Members` here?

Reply:
Sorry, I just noticed this when I was going through the code and it occurred to me that you could never initially install anything in another namespace with this logic, so it could only apply to updates, which also means the issue being addressed here could only be seen on an update, not a fresh install.